jTEI_article_2025.xml

```diff
@@ -241,7 +235,7 @@
           <head>Mass conversion approach</head>
           <p>Our goal for the mass conversion approach was to
           programatically convert all of TEI-L from its inception up
-          to 2024-04-29 into some form of simple, basic TEI using
+          to <date when="2024-04-29">April 29, 2024</date> into some form of simple, basic TEI using
```

Comment:
Oh dear. To be consistent with the rest of the document this should be “Mon 29 Apr 24”. But JTEI says to use Chicago Manual of Style, which is scizophrenic about this. They say the month-day-year order used in USA is recommended, but suggest elsewhere that day-month-year is better.

Reply:
@syd Nope--you sometimes have used the 4-digit year in the text node. I have no idea why you're using the day of the week, which is not a requirement of any style guide I know. I am changing your 2-digit years to 4-digit years across the document because I don't like the imprecision of them.

Reply:
I added the day of the week and changed the order so it's dd-mm-yyyy, and updated all to show the 4-digit year in the text node.